src/store/index.js

```diff
@@ -68,7 +69,7 @@ const store = new Vuex.Store({
   state: {
     ...emptyPans(),
     logs: [],
-    visiblePans: ['html', 'js', 'output'],
+    visiblePans: visiblePans || ['html', 'js', 'output'],
```

Comment:
we can use `$store.dispatch('showPans', [])` and `$route.query` instead

Reply:
`const params2 = router.match(window.location).query` returns nothing. Otherwise $route object isn't accessible outside vue. As for the store, I can't import it, to begin with. Given that this is not reactive and is state initialization, it doesn't have to interact with the store formally.